scripts/configure/zowe-configure-scripts.sh

```diff
@@ -3,7 +3,7 @@ sed -e "s#{{stc_name}}#${ZOWE_SERVER_PROCLIB_MEMBER}#" \
   "${ZOWE_ROOT_DIR}/scripts/templates/zowe-start.template.sh" \
   > "${ZOWE_ROOT_DIR}/scripts/zowe-start.sh" 
 
-sed -e "s#{{stc_name}}#${ZOWE_SERVER_PROCLIB_MEMBER}#" \
+sed -e "s#{{zowe_prefix}}#${ZOWE_PREFIX}#" \
```

Comment:
You aren't replacing {{stc_name}}, so I don't see how this will work as zowe-start will be left invalid?

Reply:
zowe-start.sh uses {{stc_name}} as the PROCLIB member name to start, but its JOBNAME is now {{zowe_prefix}}SV1.  When you come to stop it, zowe-stop.sh CANCELs it using its jobname of {{zowe_prefix}}SV1.

Reply:
Sorry, my comment wasn't clear. In zowe-start.template.sh you are using both stc_name and zowe_prefix `S {{stc_name}},SRVRPATH='"$ZOWE_ROOT_DIR"'",JOBNAME={{zowe_prefix}}SV1` however you are only sed injecting stc_name, so how are both values replacing during the configure?